code/game/gamemodes/cult/narsie.dm

```diff
@@ -43,8 +42,22 @@ var/global/list/narsie_list = list()
 /obj/singularity/narsie/large/New()
 	..()
 	if(announce)
-		to_world("<font size='15' color='red'><b>[uppertext(name)] HAS RISEN</b></font>")
+		//to_world("<font size='15' color='red'><b>THE [uppertext(name)] HAS RISEN</b></font>")
```

Comment:
why comment it out?

Reply:
because we already to_world when we check the amount of nar'sies in the world